source/CIPlanetProgressNode.cpp

```diff
@@ -0,0 +1,105 @@
+//
+//  CIPlanetProgressNode.cpp
+//  CoreImpact
+//
+//  Created by William Long on 4/19/21.
+//  Copyright © 2021 Game Design Initiative at Cornell. All rights reserved.
+//
+
+#include "CIPlanetProgressNode.h"
+
+#define SPF .01 //seconds per frame
```

Comment:
60fps is 0.01667, if we want more precision. 0.01 is 100fps.

Reply:
oh this is mainly so the animation gets updated every frame but also lets us change it easily